syntax = "proto3";
import "PacketsId.proto";

package my.proto.package;

message ConnectionRequest
{
	option (package_id) = CONNECTION_REQUEST;
}

message ConnectionResponse
{
	option (package_id) = CONNECTION_RESPONSE;
	string 				ip = 1;
	uint32				port = 2;
}
